provisioning/transport/amqp/src/Microsoft.Azure.Devices.Provisioning.Transport.Amqp.csproj

```diff
@@ -13,6 +21,7 @@
 
   <ItemGroup>
     <!-- FXCop -->
+    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.0.6" />
```

Comment:
Please add a TODO: + issue that we should be switching to the latest Amqp version (right now it's not working for the client so we can't since we can expect both Provisioning and IoT Hub client in the same project).